crypto/src/error.rs

```diff
@@ -249,6 +249,9 @@ pub enum CryptoError {
     /// Not supported for the moment
     #[error("Not supported for the moment")]
     Unsupported,
+    /// Invalid Context. This context was finished and cannot be further used
+    #[error("This context was already finished and cannot be further used")]
```

Comment:
"cannot be further used" -> "can no longer be used" (More natural phrasing)
```suggestion
    /// Invalid Context. This context has been finished and can no longer be used.
    #[error("This context has already been finished and can no longer be used.")]
```